contracts/SafeBridgeModule.sol

```diff
@@ -56,7 +54,7 @@ contract SafeBridgeModule {
         require(msg.sender == address(amb), "Unauthorized amb");
         require(amb.messageSourceChainId() == chainId, "Unauthorized chainId");
         require(amb.messageSender() == owner, "Unauthorized owner");
-        require(executedBridgeTransactions[amb.messageId()] == false, "Transaction already executed");
+        require(amb.messageCallStatus(amb.messageId()) == false, "Transaction already executed");
```

Comment:
I would rather remove this line at all, since it looks like a tautology for me. I don't see a way that message status can be different from false, until the execution is finished. And since the message id is unique, it can be used only once, as ensured internally in the AMB.

Reply:
Ah yeah, that totally makes sense.
I've updated the PR to include this suggestion.